docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.6
+
+RUN apt-get update
+RUN apt-get install -y ffmpeg mediainfo
+
+RUN wget $(curl --silent https://api.github.com/repos/DakaraProject/dakara-server/releases/latest | grep browser_download_url | sed -e 's/^.*\": \"//' -e 's/\"$//') -O dakara-server.zip
+
+RUN unzip dakara-server.zip
+RUN rm dakara-server.zip
+RUN mv dakara-server* /dakara-server
```

Comment:
Please put commands to get the server release in a single `get_dakara.sh` script.